README.md

```diff
@@ -168,8 +168,8 @@ return [
     ],
     'validators' => [
         'factories' => [
-            Validator\DbRecordExists::class => Validator\Db\RecordExistsFactory::class,
-            Validator\DbNoRecordExists::class => Validator\Db\NoRecordExistsFactory::class,
+            'ZF\ContentValidation\Validator\DbRecordExists' => Validator\Db\RecordExistsFactory::class,
+            'ZF\ContentValidation\Validator\DbNoRecordExists' => Validator\Db\NoRecordExistsFactory::class,
```

Comment:
@AdilHoumadi @MichaelGooden maybe you'll be surprised, but `::class` works even if class does not exist...

https://3v4l.org/FFfE1